pyproject.toml

```diff
@@ -4,19 +4,22 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "llama-extract"
-version = "0.0.5"
-description = "Infer schema and extract data from unstructured files"
-authors = ["Logan Markewich <logan@llamaindex.ai>"]
+version = "0.1.0"
+description = "Structured data extraction from files."
+authors = ["Neeraj Pradhan <neeraj@llamaindex.ai>"]
 license = "MIT"
 readme = "README.md"
 packages = [{include = "llama_extract"}]
 
 [tool.poetry.dependencies]
-python = ">=3.8.1,<4.0"
-llama-index-core = "^0.11.0"
-llama-cloud = ">=0.1.0"
+python = ">=3.10,<4.0"
+llama-index-core = "^0.12.0"
+llama-cloud = { git = "https://github.com/run-llama/platform.git", subdirectory = "llama-cloud", rev = "0aea21598587f134886895d755529cf41f0dd751" }
```

Comment:
Pinning to this rev for now, will move to released version when cutting a release.